### PR TITLE
feat(apps/interface): add "Show More" in information card

### PR DESCRIPTION
**Pull request by math-marcellino**
1. Add `Show more` link button in information card
2. Add testing coverage for `FuseLeveragedTokenInfoCardDescription` component

### CODE REVIEW

File: apps/interface/components/FuseLeveragedTokenInfoCard/Description.tsx [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+            >
+                {symbol} ({name}) is a derivatives product with no margin or
+                liquidation risks. It provides you with leveraged exposure to
+                the underlying asset, which may amplify profitability and
+                potential losses.{" "}
+                <Link
+                    data-testid="FLTInfoCardShowMoreDescLink"
+                    onClick={() => setShowMore(true)}
+                    as="button"
+                    color={gray12}
```
Comment (pyk) [inline apps/interface/components/FuseLeveragedTokenInfoCard/Description.tsx:33]: Remove `color` to <Text> below

File: apps/interface/components/FuseLeveragedTokenInfoCard/Description.tsx [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+                    display={showMore ? "none" : "inline"}
+                >
+                    Show more...
```
Comment (pyk) [inline apps/interface/components/FuseLeveragedTokenInfoCard/Description.tsx:36]: Use `<Text>` add color here and use semibold font

**Review by pyk — CHANGES_REQUESTED** [review]
@math-marcellino smol nits

**Comment by pyk** [comment]
@math-marcellino please also add case where "Show Less" is clicked

File: apps/interface/components/FuseLeveragedTokenInfoCard/Description.tsx (hunk 2 above)
Comment (math-marcellino) [reply apps/interface/components/FuseLeveragedTokenInfoCard/Description.tsx:36]: Got it sir, lemme fix it👍

**Review by pyk — APPROVED** [review]
Nice @math-marcellino

**Pull request merged**